app/views/mods_panel.py

```diff
@@ -1331,9 +1331,9 @@ def eventFilter(self, source_object: QObject, event: QEvent) -> None:
                                 mod_metadata = (
                                     self.metadata_manager.internal_local_metadata[uuid]
                                 )
-                                if not mod_metadata[
+                                if mod_metadata[
```

Comment:
Is there a potential key error for these?

Reply:
None at the moment
I am only fixing whatever PEP8 issues Pycharm is complaining about at the moment

Reply:
if you want i will undo it ?

Reply:
The difference is in the way the first condition is written:
mod_metadata["data_source"] != "expansion" is a more direct way of checking if the value is not equal to "expansion".
not mod_metadata["data_source"] == "expansion" first checks if the value is equal to "expansion", and then negates the result using the not operator.

Reply:
I agree that this definitely reads better with Lion's change.. It's really a matter of semantics though, and if it helps us be able to read the code better, I am all for it!